src/status_im/communities/core.cljs

```diff
@@ -99,10 +98,10 @@
           (assoc-in db [:communities/my-pending-requests-to-join community-id] request)
           (and (= constants/community-request-to-join-state-accepted state) (not deleted))
           (do (re-frame/dispatch [:toasts/upsert
-                                  {:icon       :i/correct
-                                   :id         :joined-community
-                                   :icon-color (:positive-01 @colors/theme)
-                                   :text       (i18n/label :t/joined-community {:community name})}])
+                                  {:icon :i/correct
```

Comment:
@codemaster115 do you know does the icon ever change? or is it always `:i/correct` for type `:positive`, `:i/info` for `:negative` etc? would be nice to encapsulate that in the component if so 👍